stable/sumologic-fluentd/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sumologic-fluentd
-version: 1.1.1
+version: 1.2.0
```

Comment:
Shouldn't a change to the supported k8s version result in a major bump?

Reply:
updated